website/docs/gitops-run/get-started.mdx

```diff
@@ -89,7 +89,8 @@ Before you continue, make sure `kubectl get nodes` returns a node which is `Read
 
 You need to set up a Git repository to put your GitOps manifests
 in. Any Git repository would do, for example create a new
-[github](https://github.com/new) repository and clone that.
+[github](https://github.com/new) repository and clone that. We use `podinfo-gitops-run`
+as the repository name in this guide. GitHub also has a command line tool called `gh` which you can use to create a repository.
```

Comment:
```suggestion
as the repository name in this guide.
```
I don't think mentioning `gh` improves the usability of this page. We should keep it as concise as possible, especially as the page is called "getting started".